aptos-node/src/network.rs

```diff
@@ -410,10 +450,15 @@ fn register_client_and_service_with_network<
     network_id: NetworkId,
     network_config: &NetworkConfig,
     application_config: NetworkApplicationConfig,
+    no_parallel: bool,
 ) -> ApplicationNetworkHandle<T> {
     let (network_sender, network_events) = network_builder.add_client_and_service(
         &application_config,
-        network_config.max_parallel_deserialization_tasks,
+        if no_parallel {
```

Comment:
Curious: why can't we support parallel deserialization, here? Ordering?

Reply:
yes, there's strong assumption on the message ordering being published right now